app/frontend/src/components/constants.ts

```diff
@@ -5,3 +5,218 @@ export const INVALID_FILE = "Invalid file.";
 export const SELECT_AN_IMAGE = "Select an image";
 export const UPLOAD_PENDING_ERROR =
   "Confirm or cancel the picture upload before saving.";
+
+//these come from https://github.com/unicode-org/cldr-json/tree/master/cldr-json/cldr-dates-modern/main
+//but really https://stackoverflow.com/a/9893752
+export const dateFormats = {
```

Comment:
I tried using "L" with the LocalizedFormats plugin, but it doesn't work with typing (it thinks the mask is _ so you can only type a single digit)